src/main/java/org/sonar/plugins/stash/IssuePathResolver.java

```diff
@@ -0,0 +1,7 @@
+package org.sonar.plugins.stash;
+
+import org.sonar.api.issue.Issue;
+
+public interface IssuePathResolver {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "IssuePathResolver" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1609)